README.md

```diff
@@ -13,80 +13,36 @@ Angular package.
 [applications]: https://news.dartlang.org/2016/03/the-new-adwords-ui-uses-dart-we-asked.html
 [AngularDart Components Gallery]: https://dart-lang.github.io/angular_components/
 
-This is a continually growing set of widgets. Recent additions include:
+These packages provide the components as well as code generation for the
```

Comment:
- These -> The following(?)
- Further above:
  - for projects using the Angular package -> for projects using [AngularDart]

Reply:
Nice cleanup.